kotlin/usecases/serverless_rds/Readme.md

```diff
@@ -827,819 +633,373 @@ The **SendMessage** class uses the AWS SDK for Kotlin SES API to send an email m
-
- ```javascript	
-	$(function() {
-
-         $( "#dialogtemplate2" ).dialog();
-
-         $('#myTable').DataTable( {
-          scrollY:        "500px",
-          scrollX:        true,
-          scrollCollapse: true,
-          paging:         true,
-          columnDefs: [
-            { width: 200, targets: 0 }
-          ],
-         fixedColumns: true
-        } );
-
-        var table = $('#myTable').DataTable();
-        $('#myTable tbody').on( 'click', 'tr', function () {
-        if ( $(this).hasClass('selected') ) {
-            $(this).removeClass('selected');
-        } else {
-            table.$('tr.selected').removeClass('selected');
-            $(this).addClass('selected');
-        }
-       } );
+### WorkItem class
 
-      // Disable the report button
-      $('#reportbutton').prop("disabled",true);
-      $('#reportbutton').css("color", "#0d010d");
+The following Kotlin code represents the **WorkItem** class.   
 
-     });
-
-
-      function modItem() {
-        var id = $('#id').val();
-        var stat = $('#status').val();
-
-        if (id == "") {
-            alert("Please select an item from the table");
-            return;
-        }
+```kotlin
+    package com.example.demo
 
-        if (stat.length > 350) {
-            alert("Status has too many characters");
-            return;
-        }
+class WorkItem {
+    var id: String? = null
+    var name: String? = null
+    var guide: String? = null
+    var date: String? = null
+    var description: String? = null
+    var status: String? = null
+}
+```
+
+## Run the application 
+
+Using the IntelliJ IDE, you can run your Spring REST API. The first time you run it, choose the run icon in the main class. The Spring API supports the following URLs. 
+
+- /api/items/{state} - A GET request that returns all active or archive data items from the **Work** table. 
+- /api/mod/{id} - A PUT request that converts the specified data item to an archived item. 
+- /api/add - A POST request that adds a new item to the database. 
+- /api/report/{email} - A PUT request that creates a report of active items and emails the report. 
+
+**Note**: The React SPA created in the next section consumes all of these URLs. 
+
+Confirm that the Spring REST API works by viewing the Active items. Enter the following URL into a browser. 
+
+http://localhost:8080/api/items/active
+
+The following illustration shows the JSON data returned from the Spring REST API. 
+
+![AWS Tracking Application](images/browser.png)
+
+## Create the React front end
+
+You can create the React SPA that consumes the JSON data returned from the Spring REST API. To create the React SPA, you can download files from the following GitHub repository. Included in this repository are instructions on how to set up the project. Click the following link to access the GitHub location [Work item tracker web client](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/resources/clients/react/item-tracker/README.md).  
```

Comment:
Update to last sentence: 
To access the GitHub location, see [Work item tracker web client](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/resources/clients/react/item-tracker/README.md).

Reply:
Dev Writer wanted this sentence as he thought it was more clear.,